cli/foundation-models/system/distillation/conversation/README.md

```diff
@@ -0,0 +1,24 @@
+# Distillation with CLI (Conversation)
+
+## 1. Create the Job
+Ensure you have the proper setup.
+1. Run `az version` and ensure the `ml` extension is installed. `ml` version should be greater or equal to 2.32.0.
+2. If the `ml` extension is not installed, run `az extension add -n ml`
+
+Run the Distillation CLI command pointing to the .YAML file in this folder and fill out the Azure ML IDs needed:
+
+```text
+az ml job create --file distillation_conversation.yaml --workspace-name [YOUR_AZURE_WORKSPACE] --resource-group [YOUR_AZURE_RESOURCE_GROUP] --subscription [YOUR_AZURE_SUBSCRIPTION]
+```
+
+## 2. Deploy to Endpoint
+Once the job finishes running, fill out the serverless_endpoint.yaml file in this folder. The necessary information can be found by 
+1. Navigating to the `model` tab in [ml studio](https://ml.azure.com). 
```

Comment:
also we should be able to deploy using UI as well through ml studio or ai studio

Reply:
Added section giving additional info on UI deployment vs CLI deployment